apis/user_search.yaml

```
@@ -0,0 +1,852 @@
+
+
+        Please read the "Domain specific params" on the [overview page](#?route=overview) to avoid common errors.
+
+        #### Examples
+          - `?roles=publisher&domain=data.ny.gov`
+          - `?roles=publisher&roles=editor&domain=data.ny.gov`
+          - `?roles=publisher,editor&domain=data.ny.gov`
+      parameters:
+        - $ref: '#/components/parameters/roles'
+        - $ref: '#/components/parameters/domain'
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SearchResponse'
+        "400":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestError'
+        "401":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+  /catalog/v1/users?team_ids={id}&domain={domain}:
+    get:
+      summary: "Find users by team_id"
+      description: >
+        Users may belong to one or more
+        [teams](https://support.socrata.com/hc/en-us/articles/360012053653-Create-and-Manage-Teams#h_01H475EQ1ASTT15A8YE1QYXFXG)
+        on a domain. Teams are uniquely identified by a four-by-four like that used with the
+        [`ids` param](#?route=get-/catalog/v1/users-ids--4x4-).
+        To find users belonging to a given team on a given domain, use the  `team_ids` and `domain` params.
+
+
+        Please read the "Domain specific params" on the [overview page](#?route=overview) to avoid common errors.
+
+        #### Examples
+          - `?team_ids=b94q-2wqa&domain=data.ny.gov`
+          - `?team_ids=b94q-2wqa&team_ids=745n-q8i9&domain=data.ny.gov`
+          - `?team_ids=b94q-2wqa,745n-q8i9&domain=data.ny.gov`
+      parameters:
+        - $ref: '#/components/parameters/team_ids'
+        - $ref: '#/components/parameters/domain'
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SearchResponse'
+        "400":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestError'
+        "401":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+  /catalog/v1/users?q={token}:
+    get:
+      summary: "Find users by query term"
+      description: >
+        Each user has an email and display name that are broken into tokens. For example if a user Someone has email
+        someone.nifty@example.com and display name "Someone Nifty", the tokens produced will be
```

```suggestion
        someone.nifty&#64;example.com and display name "Someone Nifty", the tokens produced will be
```
openapi-explorer seems to be auto-linking the email. perhaps we can use the HTML entity so we don't get a mailto: link for this fake email?

---

oh neat trick.